source/features/playground/experiments/glasses/Glass.module.css

```
@@ -0,0 +1,1962 @@
+          var(--glass-expanded-scale)
+      )
+      calc(
+        var(--glass-screen-center-y) + var(--glass-height) * (0.084529 - 0.5) *
+          var(--glass-expanded-scale)
+      ),
+    close
+  );
+
+  --glass-height: calc(var(--glass-width) / 2.895395);
+  --glass-width: 34dvw;
+  --glass-height: calc(var(--glass-width) / 2.895395);
```

Stylelint (17.9.0)

[error] Duplicate custom property "--glass-height" (declaration-block-no-duplicate-custom-properties)

(declaration-block-no-duplicate-custom-properties)